meta/meta.go

```diff
@@ -86,11 +86,19 @@ type Meta struct {
 }
 
 // NewMeta creates a Meta in transaction txn.
-func NewMeta(txn kv.Transaction) *Meta {
+// If the current Meta needs to handle a job, jobListKey is the type of the job's list.
+func NewMeta(txn kv.Transaction, jobListKeys ...JobListKeyType) *Meta {
```

Comment:
Can we always specify the JobListKey?

Reply:
A lot of places use this function, so I use this method to handle it.
And In other packages, I think we needn't distinguish the type of `jobListKeys`.